installation/language/de-DE/joomla.ini

```
@@ -87,16 +87,16 @@ INSTL_CONNECT_DB="Datenbankverbindung einrichten"
 INSTL_INSTALL_JOOMLA="Joomla! installieren"
 
 ; Site View
-INSTL_ADMIN_EMAIL_DESC="E-Mail-Adresse des Super Benutzers."
-INSTL_ADMIN_PASSWORD_DESC="Passwort des Super Benutzer."
-INSTL_ADMIN_PASSWORD_LENGTH="Mindestens 12 Zeichen eingeben."
-INSTL_ADMIN_USER_DESC="Name des Super Benutzers."
-INSTL_ADMIN_USERNAME_DESC="Benutzername des Super Benutzers."
+INSTL_ADMIN_EMAIL_DESC="E-Mail-Adresse des Super Benutzers:"
+INSTL_ADMIN_PASSWORD_DESC="Passwort des Super Benutzer:"
+INSTL_ADMIN_PASSWORD_LENGTH="Mindestens 12 Zeichen eingeben!"
```

```suggestion
INSTL_ADMIN_PASSWORD_LENGTH="Mindestens 12 Zeichen eingeben."
```

Wir versuchen ! zu vermeiden.